packages/terra-functional-testing/tests/jest/commands/expect/toMatchReference.test.js

```diff
@@ -240,12 +260,16 @@ describe('toMatchReference', () => {
       isSameDimensions: false,
       misMatchPercentage: 0.10,
     };
+    jest.spyOn(fs, 'existsSync').mockImplementation(() => true);
+    jest.spyOn(fs, 'existsSync').mockImplementation(() => true);
```

Comment:
We can use [mockFn.mockReturnValue(value)](https://jestjs.io/docs/mock-function-api#mockfnmockreturnvalueoncevalue) as well.

Reply:
Fixed here: https://github.com/cerner/terra-toolkit/pull/839/commits/e3f1c358ea6ee77af9b877d072b9ca9f77df5871#diff-4fc55674149c158d6de22cac25bcc78653afd13a67eb1270cdd05c57b4abe9ebR263